core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -201,8 +207,22 @@ object ConsumerPerformance {
       .withRequiredArg
       .describedAs("count")
       .ofType(classOf[java.lang.Integer])
-      .defaultsTo(1)
+      .defaultsTo(1)    
+    val propsFileOpt = parser.accepts("consumer.config", "External consumer properties file.")
+      .withRequiredArg
+      .describedAs("consumer properties")
+      .ofType(classOf[java.lang.String])
+      .defaultsTo("")
+    val freeformPropsOpt = parser.accepts("consumer-properties", "Freeform consumer properties in the format key=val,key=val,key=val...")
```

Comment:
Do we need both consumer-properties and consumer.config? It seems that having one of the two is enough? If we want to keep both, perhaps we should do the same in ProducerPerformance.